templates/jlink/jlinked-app.yaml

```diff
@@ -233,3 +244,73 @@ objects:
           from:
             kind: ImageStreamTag
             name: ${APPNAME}-ubimicro:latest  # ImageStreamTag for registry.access.redhat.com/ubi9/ubi-micro
+##############################################################################
+# DeploymentConfig and Route object specs
+# DeploymentConfigs are discouraged, however the documentation sugggests to still
+# use them if features are missing from Deployments, which in our case they are.
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
```

Comment:
We should not use DeploymentConfigs They are deprecated in OCP.
https://developers.redhat.com/learning/learn:openshift:replace-deprecated-deploymentconfigs-deployments/resource/resources:convert-deploymentconfig-deployment